lib/position_taken.rb

```diff
@@ -1 +1,9 @@
-# code your #position_taken? method here!
\ No newline at end of file
+# code your #position_taken? method here!
+
```

Comment:
You can use implicit ruby returns and make the entire method the crux of your logic:

```ruby
def position_taken?(board, position)
  board[position] == "X" || board[position] == "O"
end
```